static/src/javascripts/projects/common/modules/ui/first-pv-consent-banner.js

```diff
@@ -26,10 +29,28 @@ type BindableClassNames = {
     agree: string,
 };
 
+type Links = {
+    privacy: string,
+    cookies: string,
+};
+
+const links: Links = {
+    privacy: 'https://www.theguardian.com/help/privacy-policy',
```

Comment:
Can we make the hostname configurable so this works in all environments?

Reply:
Nope, we don't have a variable for the guardian's url :(